sklearn/preprocessing/_data.py

```diff
@@ -15,7 +15,7 @@
 from scipy import stats
 from scipy import optimize
 from scipy.special import boxcox
-
+from numbers import Real
```

Comment:
this import should be in the first group of imports where we place imports from the std library

Reply:
> this import should be in the first group of imports where we place imports from the std library

Hi, I've fixed it.
Thanks !!